### DISCUSSION TITLE
[Remix] Deploy with remix-flat-routes and v2_routeConvention breaking deploy

### DISCUSSION THREAD

**Discussion by the-artifabrian** [vercel/community#3347] [Bug Report]
### Summary

Hi,

I just migrated my Remix app to v2_routeConvention and remix-flat-routes for the routes structure and I'm getting the following error when trying to deploy:

```
info  built (2s)
--
18:32:20.127 | Error: Unexpected MODIFIER at 11, expected END
```

Can anyone give me some tips please on how to fix this?

Thank you

### Steps to Reproduce

I don't have much to go on here in terms of stack trace so I can't provide additional info sadly :(

**Comment by barbinbrad** [comment]
This is a Vercel specific error ☝️

**Comment by barbinbrad** [comment]
On the off chance that anyone else has this problem, it was solved for me by changing all the files ending in `index.tsx` to end with `_index.tsx`
